### ISSUE TITLE
[Feature request] Warning in GUI if files found in /usr/local/etc/swanctl/conf.d/

### ISSUE THREAD

**Issue by jtgad** [opnsense/core#8093]
Show a warning in VPN > IPsec > Connections if any *conf files are found in /usr/local/etc/swanctl/conf.d/

**Is your feature request related to a problem? Please describe.**

/usr/local/etc/swanctl/conf.d/*.conf overrides what is configured in the GUI

**Describe the solution you like**

Show a warning in VPN > IPsec > Connections area of UI when any files are found, that what you are setting in UI may or may not be overridden by manually created config files. 

**Describe alternatives you considered**

Management of *.conf files through GUI - ability to upload/download/text-edit/disable/enable specific fragments. This would be a lot more work to implement. And probably still require the same warning anyway!

**Additional context**

The VPN documentation section "Custom configurations" is not clear about what is required. No examples are given.

**Comment by Monviech** [comment]
It would be the better approach to tell what is missing so it can be added to the GUI if possible, to avoid custom configuration files for as many users as possible.

**Comment by jtgad** [comment]
Yeah, it would be better if GUI allowed administration of every parameter. But that could take years to implement, warning could be implemented today.

**Comment by Monviech** [comment]
All of these settings do not have to be added in a big bang approach. Just be more precise what is missing for you.
